<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogCommandShowText</class>
 <widget class="QDialog" name="DialogCommandShowText">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>788</width>
    <height>413</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Show text...</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="leftMargin">
    <number>15</number>
   </property>
   <property name="topMargin">
    <number>10</number>
   </property>
   <property name="rightMargin">
    <number>15</number>
   </property>
   <property name="bottomMargin">
    <number>10</number>
   </property>
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_3" stretch="1,0">
     <property name="spacing">
      <number>15</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0,1,0">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,0">
         <item>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="labelInterlocutor">
             <property name="text">
              <string>Interlocutor:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="PanelPrimitiveValue" name="panelPrimitiveInterlocutor" native="true">
             <property name="minimumSize">
              <size>
               <width>250</width>
               <height>27</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>250</width>
               <height>27</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="labelFaceset">
             <property name="text">
              <string>Faceset:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="WidgetPicture" name="widgetPictureFaceset" native="true">
             <property name="minimumSize">
              <size>
               <width>250</width>
               <height>27</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>250</width>
               <height>27</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="2" column="0" colspan="2">
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>378</width>
               <height>0</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,0,0,0,1,1,1,1,1">
           <property name="spacing">
            <number>0</number>
           </property>
           <item>
            <widget class="QPushButton" name="pushButtonBold">
             <property name="maximumSize">
              <size>
               <width>25</width>
               <height>25</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="../../ressources.qrc">
               <normaloff>:/icons/Ressources/bold.png</normaloff>:/icons/Ressources/bold.png</iconset>
             </property>
             <property name="flat">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonItalic">
             <property name="maximumSize">
              <size>
               <width>25</width>
               <height>25</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="../../ressources.qrc">
               <normaloff>:/icons/Ressources/italic.png</normaloff>:/icons/Ressources/italic.png</iconset>
             </property>
             <property name="flat">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonLeft">
             <property name="maximumSize">
              <size>
               <width>25</width>
               <height>25</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="../../ressources.qrc">
               <normaloff>:/icons/Ressources/text-left.png</normaloff>:/icons/Ressources/text-left.png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonCenter">
             <property name="maximumSize">
              <size>
               <width>25</width>
               <height>25</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="../../ressources.qrc">
               <normaloff>:/icons/Ressources/text-center.png</normaloff>:/icons/Ressources/text-center.png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonRight">
             <property name="maximumSize">
              <size>
               <width>25</width>
               <height>25</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="../../ressources.qrc">
               <normaloff>:/icons/Ressources/text-right.png</normaloff>:/icons/Ressources/text-right.png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBoxFontSize">
             <property name="maximumSize">
              <size>
               <width>125</width>
               <height>16777215</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBoxFontName">
             <property name="maximumSize">
              <size>
               <width>125</width>
               <height>16777215</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBoxTextColor">
             <property name="maximumSize">
              <size>
               <width>125</width>
               <height>16777215</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBoxBackColor">
             <property name="maximumSize">
              <size>
               <width>125</width>
               <height>16777215</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBoxStrokeColor">
             <property name="maximumSize">
              <size>
               <width>125</width>
               <height>16777215</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0,0,0,0,0">
           <property name="spacing">
            <number>0</number>
           </property>
           <item>
            <widget class="QComboBox" name="comboBoxVariable">
             <property name="minimumSize">
              <size>
               <width>125</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>125</width>
               <height>16777215</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBoxParameters">
             <property name="minimumSize">
              <size>
               <width>125</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>125</width>
               <height>16777215</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBoxProperties">
             <property name="minimumSize">
              <size>
               <width>125</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>125</width>
               <height>16777215</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBoxHeroName">
             <property name="minimumSize">
              <size>
               <width>125</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>125</width>
               <height>16777215</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBoxIcon">
             <property name="minimumSize">
              <size>
               <width>125</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>125</width>
               <height>16777215</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>248</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="plainTextMessage"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PanelPrimitiveValue</class>
   <extends>QWidget</extends>
   <header>panelprimitivevalue.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WidgetPicture</class>
   <extends>QWidget</extends>
   <header>widgetpicture.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../ressources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogCommandShowText</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogCommandShowText</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
